### ISSUE TITLE
BackendAllocatorException in dfc haf2hef conversion   

### ISSUE THREAD

**Issue by DCC-lzhy** [hailo-ai/hailo_model_zoo#113]
when I run 
![image](https://github.com/user-attachments/assets/96745995-40ff-40f3-bfa8-86730741c519)

`hailodfc3.28.0/lib/python3.8/site-packages/hailo_tools/build/compiler: symbol lookup error: /home/hzhy/miniconda3/envs/hailodfc3.28.0/lib/python3.8/site-packages/hailo_tools/build/compiler: undefined symbol: _ZN6gflags14FlagRegistererC1INSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEEEPKcS9_S9_PT_SB_`

`---------------------------------------------------------------------------
BackendAllocatorException                 Traceback (most recent call last)
Cell In[4], line 1
----> 1 hef = runner.compile()
      3 file_name = f"{model_name}.hef"
      4 with open(file_name, "wb") as f:`

I install 3.28 dfc in virtual system "Ubuntu 20.04.6 LTS (GNU/Linux 5.15.0-107-generic x86_64)" in VMware software, the onnx to haf/quantized har is ok, but quantized har to hef file failed.

In other words, I guess it needs hailo hardware to compile. I have put hailo8 hardware in the arm development board, so how can i convert quantized har file to hef file under this situation?

I hope get your answers. It's been bothering me for a long time.

**Comment by DCC-lzhy** [comment]
it comes from hailodfc3.28.0/lib/python3.8/site-packages/hailo_tutorials/notebooks/DFC_3_Compilation_Tutorial.ipynb which from  installed hailo_dataflow_compiler-3.28.0-py3-none-linux_x86_64.
